### DISCUSSION TITLE
Select form field Limit

### DISCUSSION THREAD

**Discussion by mrbarletta** [Budibase/budibase#4800] [Help] [answered]
Hi team, is there a limit to the Select field?  I can't display a list of countries, tried a JSON online and a local DB table and both only shows some of the records.

I inspected the network request and the datasource XHR returns ALL the data.

thanks

**Comment by mjashanks** [comment, marked as answer]
If you are populating the select field using a data provider, then you might need to increase the `limit` on the data provider.

**Reply by mrbarletta** [reply]
That was it, thanks!
